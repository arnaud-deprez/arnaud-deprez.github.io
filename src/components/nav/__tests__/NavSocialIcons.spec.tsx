import React from 'react'
import { render } from '@testing-library/react'
import { NavSocialIcons } from '..'

describe('<NavSocialIcons/>', () => {
  it('renders correctly', () => {
    const { asFragment } = render(
      <NavSocialIcons
        linkedin="https://linkedin.com"
        github="https://github.com"
        twitter="https://twitter.com"
      />
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
