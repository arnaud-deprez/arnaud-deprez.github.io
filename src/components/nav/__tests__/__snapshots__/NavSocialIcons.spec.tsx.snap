// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavSocialIcons/> renders correctly 1`] = `
<DocumentFragment>
  <ul
    class="social-icons nav"
    role=""
  >
    <li
      class="nav-item"
    >
      <a
        aria-label="Linked'In"
        class="nav-link"
        data-rr-ui-event-key="https://linkedin.com"
        href="https://linkedin.com"
        rel="noopener noreferrer"
        target="_blank"
      />
    </li>
    <li
      class="nav-item"
    >
      <a
        aria-label="Github"
        class="nav-link"
        data-rr-ui-event-key="https://github.com"
        href="https://github.com"
        rel="noopener noreferrer"
        target="_blank"
      />
    </li>
    <li
      class="nav-item"
    >
      <a
        aria-label="Twitter"
        class="nav-link"
        data-rr-ui-event-key="https://twitter.com"
        href="https://twitter.com"
        rel="noopener noreferrer"
        target="_blank"
      />
    </li>
  </ul>
</DocumentFragment>
`;
